services/hub/README.md

```diff
@@ -0,0 +1,14 @@
+# Hub Service
+
+The hub service provides `server sent events` (`sse`) functionality to clients
+
+## Subscribing
+
+A client can use the `hub/sse` endpoint to subscribe to events. It will open an http connection for the server to send events.
+These events can inform the client about different changes on the server, for example: file uploads, shares, space memberships, ...
+See `Available Events` for details
+
+## Available Events
+For complete and up-to-date list of available events see `pkg/service/events.go`
+
+For starters the `hub` service only serves the `UploadReady` event which is emitted when a files postprocessing is finished and the file is ready to work with
```

Comment:
```suggestion
A client can use the `hub/sse` endpoint to subscribe to events. This will open a http(s) connection for the server to send events to subscribed clients. These events can inform clients about various changes on the server like: file uploads, shares, space memberships, etc.

## Available Events
For a complete and up-to-date list of available events see `pkg/service/events.go`.

Note that for the time being, the `hub` service only serves the `UploadReady` event which is emitted when postprocessing a file has finished and the file is available for user access.
```